gradio/routes.py

```diff
@@ -257,14 +263,26 @@ def create_app(
         app_kwargs["lifespan"] = create_lifespan_handler(
             app_kwargs.get("lifespan", None), *delete_cache
         )
-        app = App(auth_dependency=auth_dependency, **app_kwargs)
+        app = App(auth_dependency=auth_dependency, **app_kwargs, debug=True)
+        router = APIRouter(prefix="/gradio_api")
```

Comment:
I think we should add a constant in `route_utils` and use this here and in the other files. Other than that, this approach of using the router lgtm!